src/Framework/test/SharedFxTests.cs

```diff
@@ -155,6 +159,48 @@ public void ItContainsValidDepsJson()
             }
         }
 
+        [Fact]
+        public void SharedFrameworkAssembliesHaveExpectedAssemblyVersions()
+        {
+            // Only test managed assemblies
+            IEnumerable<string> dlls = Directory.GetFiles(_sharedFxRoot, "*.dll", SearchOption.AllDirectories).Where(i => !i.Contains("aspnetcorev2_inprocess"));
+            Assert.NotEmpty(dlls);
+
+            Assert.All(dlls, path =>
+            {
+                var fileName = Path.GetFileNameWithoutExtension(path);
+                var assemblyName = AssemblyName.GetAssemblyName(path);
```

Comment:
Where are these two `var`s used❔ Remove if I'm reading this correctly.

Reply:
Nowhere, vestigial from copying stuff from a different test